tests/run-test.py

```diff
@@ -41,12 +41,18 @@ def exec_func(cmd):
     err = p.close()
     return output, err
 
+# translate string to unescaped version in shell environment
+# we only need to consider '$', '`' and '\'
+# ref: https://www.gnu.org/software/bash/manual/html_node/Double-Quotes.html
+def to_unescaped_str(cmd):
```

Comment:
how about add tests for usage of '$' and '\' in the test?

Reply:
sure